frontend/src/components/Header.js

```diff
@@ -204,6 +204,7 @@ class Header extends Component {
                                 <NavLink to="/product#rings" className={this.setClassName("#rings", "")}>Rings</NavLink>
                                 <NavLink to="/product#necklaces" className={this.setClassName("#necklaces", "")}>Necklaces</NavLink>
                                 <NavLink to="/product#earrings" className={this.setClassName("#earrings", "")}>Earrings</NavLink>
+                                <NavLink to="/custom" className={this.setClassName("#custom", "")}>Custom Creations</NavLink>
```

Comment:
You have to add it also in the case that a user is signed in, because now if I am signed in as a normal user I cannot see this choice. But do not add it in admin.